packages/data-stores/src/onboard/reducer.ts

```diff
@@ -338,6 +345,7 @@ const reducer = combineReducers( {
 	storeType,
 	selectedFonts,
 	selectedDesign,
+	isGeneratedDesignsView,
```

Comment:
I don't think it's worth introducing a new state at this time as we're ready to deploy the UDP in the next few days. Hence, I'd prefer to add an independent `useEffect` in `site-setup-design-picker.tsx` to toggle the progress bar class directly whenever the `showGenerated is changed

Reply:
The progress bar is not child of the `DesignSetup`, therefore I have to use store to identify the current generated designs view, what do you think?

Reply:
To simplify the work, I can revert the `ONBOARDING` store change and use `STEPPER_INTERNAL_STORE` instead for the Generated Designs view flag, will it work?